tests/main/interfaces-netlink-connector/task.yaml

```diff
@@ -12,14 +12,15 @@ prepare: |
     install_local test-snapd-netlink-connector
 
 execute: |
-    CONNECTED_PATTERN=":netlink-audit +test-snapd-netlink-connector"
+    CONNECTED_PATTERN=":netlink-connector +test-snapd-netlink-connector"
```

Comment:
This looks like an unrelated fix. If this branch doesn't come in, can you please cherry pick it and propose separately.

Reply:
sure